test/functional/neutronless/bigip_interaction.py

```diff
@@ -176,19 +173,13 @@ def __collect_diff(cls, test_method):
         try:
             cls.__check_results(result)
         except RuntimeError:
-            raise AssertionError(diff_file)
+            if os.path.getsize(diff_file) != os.path.getsize(sesson_file):
```

Comment:
Shouldn't this test (of the file sizes) be part of "__check_results"?

Reply:
no.  The function `__check_results` is used as a means of checking all shell command's responses, not just for the diff shell command execution.  I would rather this guy remain as simple as he is.